app/ide-desktop/lib/dashboard/src/text/english.json

```diff
@@ -306,6 +306,7 @@
   "openInfoMenu": "Open info menu",
   "noProjectIsCurrentlyOpen": "No project is currently open.",
   "offlineTitle": "You are offline",
+  "offlineErrorMessage": "It seems like you're offline. Please make sure you're connected to the internet and try again",
```

Comment:
contractions like "you're" tend to sound informal, prefer the expanded form ("you are")

Reply:
Thanks!